packaging/ubuntu-14.04/rules

```diff
@@ -55,7 +55,7 @@ endif
 # check if we need to include the testkeys in the binary
 TAGS=
 ifneq (,$(filter testkeys,$(DEB_BUILD_OPTIONS)))
-	TAGS=-tags withtestkeys
```

Comment:
Note, oportunistic fix.. I think we had a subtle bug here (and also below in other rules file) - in override_dh_auto_test below `-tags` was added too, go was just lenient and ignored one of them.